interfaces/builtin/greengrass_support.go

```diff
@@ -128,11 +128,12 @@ mount options=(rw, bind) /dev/null -> /proc/sched_debug,
 mount options=(rw, bind) /dev/null -> /proc/timer_stats,
 
 # perform the pivot_root into the overlay
-pivot_root oldroot=/var/snap/greengrass/x1/rootfs/.pivot_root*/ /var/snap/greengrass/*/rootfs/,
+pivot_root oldroot=/var/snap/greengrass/@{SNAP_REVISION}/rootfs/.pivot_root*/ /var/snap/greengrass/*/rootfs/,
```

Comment:
Are you aware of the consequences of using pivot_root and how it makes most of our apparmor rules ineffective?

(Sorry for pulling this out now, I didn't notice this before)